paper/paper.Rmd

```diff
@@ -0,0 +1,149 @@
+---
+title: "Static Code Analysis for R"
+date: "`r Sys.Date()`"
+tags: ["R", "linter", "tidyverse"]
+
+authors:
+  - name: Jim Hester
+    affiliation: 1
+    orcid: ~
+
+affiliations:
+  - index: 1
+    name: Netflix
+
+output: 
+  rticles::joss_article:
+    keep_tex: true
+bibliography: paper.bib
+csl: apa.csl
+journal: JOSS
+link-citations: yes
+---
+
+```{r, warning=FALSE, message=FALSE, echo=FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  out.width = "100%",
+  dpi = 300,
+  comment = "#>"
+)
+
+options(width = 60)
+```
+
+# Statement of Need
+
+The R programming language [@base2023] is a popular choice for statistical analysis and visualization, and is used by a wide range of researchers and data scientists. The `{lintr}` package is an open-source R package that provides static code analysis to check for a variety of common problems related to readability, efficiency, consistency, style, etc. It is designed to be easy to use and integrate into existing workflows, and can be run from the command line or used as part of an automated build or continuous integration process. `{lintr}` also integrates with a number of popular IDEs and text editors, such as RStudio and Visual Studio Code, making it convenient for users to run `{lintr}` checks on their code as they work.
```

Comment:
if this audience may not be familiar with R, it's important to clarify that R is dynamically-typed, since that colors so much of what we can/can't do with {lintr}.

Reply:
So far, I don't see any mention that a primary purpose of {lintr} is helping to enforce the tidyverse style guide in particular. I think this is a useful contextual touchstone.

Reply:
> So far, I don't see any mention that a primary purpose of {lintr} is helping to enforce the tidyverse style guide in particular. I think this is a useful contextual touchstone.

@MichaelChirico It's a bit ironic that (AFAICT) none of the tidyverse packages actually use `{lintr}` in their development workflow 😅